### PR TITLE
Chore: Removes fs beta program mentions

### PR DESCRIPTION
**Pull request by eduardoformiga**
This PR aims to remove the mentions to Faststore Beta program.

### CODE REVIEW

File: blog/2022-02-08-webops.md [hunk 1]
```diff
@@ -12,4 +12,4 @@ Under high loads, workflows that used to take from 2 to 10 minutes started to sp
 
 Earlier this week, we began an investigation into it, and we managed to improve the situation drastically. **Our pipelines runtime is now back to the 2-10 minutes range, depending on the workload type and size of the store.**
 
-We are working hard to deliver improvements to our platform such as this. Expect further improvements during the closed beta as we work on our platform's stability and performance!
```
Comment (gvc) [inline blog/2022-02-08-webops.md]: I think this can stay.

**Review by gvc — COMMENTED** [review]
Just a small change. There's no problem referring to the beta program in a blog post. Otherwise 💯

File: blog/2022-02-08-webops.md [hunk 2]
```diff
@@ -12,4 +12,4 @@ Under high loads, workflows that used to take from 2 to 10 minutes started to sp
...
+We are working hard to deliver improvements to our platform such as this. Expect further improvements as we work on our platform's stability and performance!
```
Comment (eduardoformiga) [inline blog/2022-02-08-webops.md]:
```suggestion
We are working hard to deliver improvements to our platform such as this. Expect further improvements during the closed beta as we work on our platform's stability and performance!
```

File: blog/2022-02-08-webops.md (hunk 1 above)
Comment (eduardoformiga) [reply blog/2022-02-08-webops.md]: Nice. Done [here](https://github.com/vtex/faststore-docs/pull/38/commits/a2b3302555cea36cd0c1d503e0fca75afcdd6ad5).

**Pull request merged**